plugins/pulp_docker/plugins/importers/sync.py

```diff
@@ -306,5 +306,5 @@ def _move_file(self, unit):
         :param unit: a pulp unit
         :type  unit: pulp.plugins.model.Unit
         """
-        _logger.debug('moving files in to place for Unit {}'.format(unit))
+        _logger.debug('moving files in to place for Unit {unit}' % format(unit=unit))
```

Comment:
This is confusing the two interpolation methods. There should be no percent signs.

```python
_logger.debug('moving files in to place for Unit {unit}'.format(unit=unit))
```